pkg/test/runner/runner.go

```diff
@@ -114,7 +118,10 @@ func (r *Runner) runPipeline() error {
 	if err != nil {
 		return fmt.Errorf("failed to create temporary dir: %w", err)
 	}
-	defer os.RemoveAll(tmpDir)
+	if !r.testCase.Config.Debug {
+		// if debug is true, keep the test directory around for debugging
+		defer os.RemoveAll(tmpDir)
```

Comment:
It's better to output the path of temporary directory

Reply:
Good pt. Added.